sdk-lib/src/androidTest/java/cash/z/ecc/android/sdk/fixture/WalletFixture.kt

```diff
@@ -0,0 +1,18 @@
+package cash.z.ecc.android.sdk.fixture
+
+import cash.z.ecc.android.bip39.Mnemonics
+import cash.z.ecc.android.sdk.model.Account
+import cash.z.ecc.android.sdk.model.ZcashNetwork
+import cash.z.ecc.android.sdk.tool.DerivationTool
+
+object WalletFixture {
+    val NETWORK = ZcashNetwork.Mainnet
+    const val SEED_PHRASE =
+        "kitchen renew wide common vague fold vacuum tilt amazing pear square gossip jewel month tree shock scan alpha just spot fluid toilet view dinner"
```

Comment:
In my review, I didn't spot where this was used (or a corresponding deletion of this seed phrase from a test). What motivated its addition?

Reply:
We need hard-coded fixture values to write some unit tests, and this is consistent with a pattern we are adopting in the Secant app.

These are used by the unit tests for DerivationTool and the UnifiedSpendingKey.